### ISSUE TITLE
Hover Documentation does not display the icon

### ISSUE THREAD

**Issue by FFTU** [KamasamaK/vscode-cfml#64]
Hover Documentation does not display the icon
![QQ截图20210813175842](https://user-images.githubusercontent.com/27883482/129340567-6b7568bb-7150-458c-80d0-0c883596ddb6.jpg)
![QQ截图20210813180035](https://user-images.githubusercontent.com/27883482/129340578-53d1c32a-c674-45d6-b51b-a6f6d727d121.jpg)

How should I solve this problem, thank you

**Comment by KamasamaK** [comment]
This is a bug in VS Code and should supposedly be resolved in v1.60. Thankfully, the alt text should be sufficient for now.

Ref: microsoft/vscode#119786

**Comment by KamasamaK** [comment]
Confirmed resolved in VS Code v1.60.
